backend/accounts.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backend
```

Comment:
Not sure why separate file is better, not enough files named account*.go? :)

```
$ find . -name account*.go
./backend/accounts/account.go
./backend/coins/eth/account.go
./backend/coins/btc/account_test.go
./backend/coins/btc/account.go
./backend/config/accounts.go
./vendor/github.com/ethereum/go-ethereum/accounts/keystore/account_cache.go
./vendor/github.com/ethereum/go-ethereum/accounts/accounts.go
```

I think the only `filterAccounts` func here would be happy in backend.go, too.

Reply:
I will add more functions, and possibly move over more of the account handling functions here, so that backend.go is smaller and and easier to read.

Reply:
The confusing part to me is that there's also `backend/accounts` package.